### DISCUSSION TITLE
Lab 1 Part 2 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [shivanidoshi26/Interactive-Lab-Hub#7] [General]
Add your feedback as a comment to this thread. Thanks!

**Comment by snlee159** [comment]
My thoughts on your part 2 device were:
- I loved the sound effects. They were very relatable
- The choice to switch from light green to yellow for not ripe things makes a lot of sense and made things much clearer
- Having the words "rotten", "not ripe", etc appear on the screen could make things even more clear

**Comment by Rpoddar1953** [comment]
- I really like the voice, it makes things a lot clearer!
- I feel like with the sound effect there should be an option to increase or decrease the level of the voice, to make the experience better for people who might have trouble hearing or so that its not too loud.

**Comment by singhaniasnigdha** [comment]
- Physical Device: The costume with all the buttons was a great improvement! And the image clearly depicts the purpose of all ports and buttons, great eye for detail! Would've loved to see the use of the buttons in the video for more context.
- Colors and Voice Prompt: The choice of colors fits the message that needs to be conveyed! And the voice is a great addition - because now your device will be so helpful for those who are visually impaired.

Interesting idea and execution!

**Comment by anam884** [comment]
Good job on adding the sound for accessibility! 

Regarding the setting of your device, in grocery stores, fruits and vegetables are usually displayed in bulk, you could have elaborated on either it requires scanning/sessing a single fruit, or can it spot the ripe one from bulk?
